.gitmodules

```
@@ -0,0 +1,3 @@
+[submodule "vue-element-admin"]
+	path = vue-element-admin
```

`git subtree` still imports the whole repo into this one so it isn't beneficial to us. This PR currently adds about 300 files but it should only change 3 or 4. I think that what we're trying to do is achievable by changing this line to:

```suggestion
	path = vue-element-admin/repo
```

Then you'll be able to create the regular `/vue-element-admin/install.sh` and make the required changes in `/vercel.json`